apps/easypid/src/features/receive/FunkeCredentialNotificationScreen.tsx

```diff
@@ -132,6 +139,12 @@ export function FunkeCredentialNotificationScreen() {
     [agent]
   )
 
+  // TODO: Should we add this to the activitiy? We also don't do it for issuance
```

Comment:
https://animosolutions.slack.com/archives/C0759K2BX0S/p1732446331079129?thread_ts=1730797262.934249&cid=C0759K2BX0S

Reply:
Okay. Reason for the comment is that i see this more as a failed issuance. But yeah if that's also on the list.

We don't support failed issuance though? And just adding this as failed proof request is weird i think as it's during issuance